### ISSUE TITLE
Feature: HEALTHCHECK

### ISSUE THREAD

**Issue by willfarrell** [elastic/beats-docker#13]
Would you like a PR for having a default `HEALTHCHECK` in place. I'm more than happy to submit one.

Filebeat example:
```Dockerfile
HEALTHCHECK --interval=5s --timeout=3s \
    CMD ps aux | grep '[f]ilebeat' || exit 1
```

Or perhaps there is a better command that does a `ping` to beats that would be more accurate?

**Comment by exekias** [comment]
A healthcheck would probably make sense, but we may need to add some instrumentation to beats for it. 

Not sure if your proposed check would work as filebeat (or any other beat) is the entrypoint, that means the container dies with it

**Comment by willfarrell** [comment]
You are correct @exekias. The above healthcheck, is more for monitoring consistency. Is it possible for filebeat to be `unhealthy` and be running (assuming the syntax of the config is fine)? Maybe we should draft out which situations would constitute `unhealthy` then look into what approach would fit best. It may be a simple as grepping the output.

**Comment by exekias** [comment]
When this PR gets through we may expose the results as an HTTP endpoint: https://github.com/elastic/beats/pull/4590
